docs/content/docs/tutorials/memorydb-example.md

```diff
@@ -0,0 +1,288 @@
+---
+title: "Deploy MemoryDB Cluster Using the ACK MemoryDB Controller"
+description: "Create managed MemoryDB Cluster from a Amazon Elastic Kubernetes Service (EKS) deployment."
```

Comment:
I see you've taken this format from the RDS example, but I dislike using the term "deployment" here because it could be confused for a K8s `Deployment` type. Also I think these tutorials could be expanded to more than just EKS, so maybe even that's unnecessary. Here is my suggestion:

```suggestion
description: "Create a managed MemoryDB Cluster using the memorydb-controller"
```

(also nit: Please remove the trailing period)

Reply:
Updated the document.